README.md

```diff
@@ -24,9 +24,15 @@ Any changes made by this module affect the current `running-config`. These chang
 
 ## Setup
 
-### Setup Requirements
+**Device access:** This module requires a user that can access the device via SSH and that has the `enable mode` privilege.
 
-This module requires a user that can access the device via SSH and that has the `enable mode` privilege.
+**Proxy Puppet agent:** Since a Puppet agent is not available for the Catalysts (and, seriously, who would want to run an agent on them?) we need a proxy Puppet agent (either a compile master, or another agent) to run puppet on behalf of the device.
```

Comment:
Suggested change: omit "(and, seriously, who would want to run an agent on them?)".